File: 4_count_A.py
s = input()
L = int(input())

max_a = 0

for i in range(0, len(s), L):
    part = s[i:i+L]

    count_a = part.count('a')

    max_a = max(max_a, count_a)

print(max_a)

File: 4_count_A.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    string str;
    int l;
    cin>>str;
    cin>>l;
    int maxA=0;
    for(int i=0;i<str.length();i+=l){
        int countA=0;
        for(int j=i;j<i+l && j<str.length();j++){
            if(str[j]=='a'){
                countA++;

            }
        }
        if(countA>maxA){
            maxA=countA;
        }
        
    }
    cout<<maxA;
    return 0;
}